encoding/descaling.md

```diff
@@ -1,477 +1,332 @@
-without dabbling with specific masks.
-However, as long as you've got
-a source with nice, clean lineart,
-descaling might be a viable option,
-and possibly nullify the need
-for a lot of other filtering.
-
-
-## Preparation
-
-To prepare for figuring out the native resolution,
-you'll want to use [getnative][],
-a Python script designed for
-figuring out the resolution
-a show was animated at.
-For the actual descaling,
-make sure to grab BluBb-mADe's [descale][].
-
-One important thing
-to keep in mind when descaling
-is that you will never find
-"the perfect descaling settings".
-Even if you find the exact settings
-that the studio used,
-you won't be able to get a frame-perfect replica
-of the original frame.
-This is because the best available sources to consumers,
-usually Blu-rays,
-aren't lossless.
-There are always going to be some differences
-from the original masters
-which makes it impossible
-to perfectly descale something.
-However, usually those differences
-are so small that they're negligible.
-If you run into a case where
-you can't find any low relative error spikes,
-descaling can be highly destructive.
-It's instead recommended to resize
-as you would normally,
-or to not mess with scaling at all.
+Among the many tools available in an encoder's arsenal,
+the ability to descale might arguably be one of the best.
+But what does descaling actually mean?
+In what way does it differ from normal downscaling?
+How do you actually descale properly?
+This guide will explain the groundwork for you
+to start descaling effectively.
+
+
+## Descaling vs. Downscaling
+
+What the difference between downscaling and descaling is
+is a question that gets asked fairly frequently.
+Simply put,
+downscaling is the act of resizing an image
+to a smaller resolution than the original image,
+while with descaling you are actively reversing the math
+applied to an upscale,
+reconstructing the original image before upscaling
+(which coincidentally also means you get a smaller image).
+
+For a more thorough explanation on the intrinsics of descaling,
+please refer to the [vapoursynth-descale Github page][vapoursynth-descale].
+
+Descaling is generally the preferred method
+of handling upscaled content whenever possible.
+When performing a simple downscale,
+information will be lost in the process.
+A proper descale will, on the other hand,
+(theoretically) preserve all the *actual* detail in an image.
+
+That said, if you can't perform a good descale,
+you will usually end up with haloing/ringing artifacting,
+as illustrated below:
+
+![](images/descale/descale_ubw_00_bad.png)
+
+<div class="warning box"><p>
+Only descale if you can be sure,
+beyond reasonable doubt,
+that you have figured out the correct resolution and kernel.
+</p></div>
+
+[vapoursynth-descale]: https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale
+
+
+# Discerning the native resolution
+
+The first step to descaling is to figure out
```

Comment:
I don't quite understand this comment. What are we supposed to put more emphasis on? That you need the correct resolution? Because I can't think of anything that puts more emphasis on it than writing an entire section on it.

Reply:
I think it's just the "first step" wording that I don't like here, because it's not just the first but the most important one.